marimo/_server/api/set_cell_config.py

```diff
@@ -17,7 +18,18 @@ class SetCellConfig:
     configs: Dict[CellId_t, Dict[str, object]]
 
 
-class SetCellConfigHandler(ValidatedHandler):
+class SetCellConfigHandler(tornado.web.RequestHandler):
+    # This mutates cell config in the kernel.
+    #
+    # NB: Checking server token?
+    #
+    # - To prevent a frontend from accidentally setting the config of a kernel
+    #   it wasn't started for (eg, frontend created locally on a port that's
+    #   forwarded to a remote server for a different marimo app), we could
+    #   require a server token.
```

Comment:
Not a big concern, since this doesn't write to disk. Just might cause a confusing bug, but as an edge case fine to ignore for now I think

Reply:
why not  extend the `EditHandler` that validate the server token? we don't use this handler is run mode afaik

Reply:
I noticed it's used in run mode before instantiating actually. I did extend EditHandler but got an error when trying marimo run, which is how I found out. I guess we shouldn't need to call this in run mode tho, as long as backend is parsing and setting the configs. Can investigate in another change